### ISSUE TITLE
`use-count` shouldn't apply to `dictionary.values()`

### ISSUE THREAD

**Issue by ronnodas** [sourcery-ai/sourcery#169]
### Issue description or question
Replacing sum with count works when the iterable is a list but not when it's a dictionary view:
```
sum(value == 10 for value in dictionary.values())
```
works fine while
```
dictionary.values().count(10)
```
is invalid.

### Sourcery Version

0.9.5

### Code editor or IDE name and version

Pycharm v2021.2.3

### OS name and version

Pop! OS 21.04

**Comment by Hellebore** [comment]
Good spot - we will address.

**Comment by Hellebore** [comment]
Fixed in 0.9.6.
